bastion/src/message.rs

```diff
@@ -675,3 +675,56 @@ macro_rules! msg {
         }
     } };
 }
+
+#[macro_export]
+/// Answers to a given message, with the given answer.
+///
+/// # Example
+///
+/// ```rust
+/// # use bastion::prelude::*;
+/// #
+/// # fn main() {
+///     # Bastion::init();
+///     # let children_ref =
+/// // Create a new child...
+/// Bastion::children(|children| {
+///     children.with_exec(|ctx: BastionContext| {
+///         async move {
+///             let msg = ctx.recv().await?;
+///             answer!(msg, "goodbye").unwrap();
+///             Ok(())
+///         }
+///     })
+/// }).expect("Couldn't create the children group.");
+///
+///     # Bastion::children(|children| {
+///         # children.with_exec(move |ctx: BastionContext| {
+///             # let child_ref = children_ref.elems()[0].clone();
+///             # async move {
+/// let answer: Answer = ctx.ask(&child_ref.addr(), "hello").expect("Couldn't send the message.");
+///
+/// msg! { answer.await.expect("Couldn't receive the answer."),
+///     msg: &'static str => {
+///         assert_eq!(msg, "goodbye");
+///     };
+///     _: _ => ();
+/// }
+///                 #
+///                 # Ok(())
+///             # }
+///         # })
+///     # }).unwrap();
+///     #
+///     # Bastion::start();
+///     # Bastion::stop();
+///     # Bastion::block_until_stopped();
+/// # }
```

Comment:
Can you post the rendered doc to pr?

Reply:
![image](https://user-images.githubusercontent.com/39732259/76170345-6caa5880-6181-11ea-9560-945b3f3b83fa.png)